### ISSUE TITLE
🐛 Parser crashes

### ISSUE THREAD

**Issue by ematipico** [biomejs/biome#5552]
### Environment information

```block
main
```

### What happened?

The [following code](https://next.biomejs.dev/playground/?code=aQBtAHAAbwByAHQAIAB7ACAAaQBuAGoAZQBjAHQALAAgAEkAbgBqAGUAYwB0AGEAYgBsAGUALAAgAGkAbgBqAGUAYwB0ACAAfQAgAGYAcgBvAG0AIAAnAEAAYQBuAGcAdQBsAGEAcgAvAGMAbwByAGUAJwA7AA%3D%3D&ruleDomains.solid=recommended) should raise a parse diagnostic, but it doesnt: 

```js
import { inject, Injectable, inject } from '@angular/core';
```

As you notice, the playground is from the **beta**. The current version of Biome, instead, raises a diagnostic correctly: https://biomejs.dev/playground/?attributePosition=multiline&code=aQBtAHAAbwByAHQAIAB7ACAAaQBuAGoAZQBjAHQALAAgAEkAbgBqAGUAYwB0AGEAYgBsAGUALAAgAGkAbgBqAGUAYwB0ACAAfQAgAGYAcgBvAG0AIAAnAEAAYQBuAGcAdQBsAGEAcgAvAGMAbwByAGUAJwA7AA%3D%3D

### Expected result

It should raise a diagnostic, or not crash and emit a bogus CST.

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by amasotti** [comment]
Interesting... if I add the ill-formed js code to the test cases (in the current main branch), it gets recognized and shows diagnostics. The CST is not empty as it is in the playground.

* CST
```
0: JS_MODULE@0..59
  0: (empty)
  1: (empty)
  2: JS_DIRECTIVE_LIST@0..0
  3: JS_MODULE_ITEM_LIST@0..59
    0: JS_IMPORT@0..59
      0: IMPORT_KW@0..7 "import" [] [Whitespace(" ")]
      1: JS_IMPORT_NAMED_CLAUSE@7..58
        0: (empty)
        1: JS_NAMED_IMPORT_SPECIFIERS@7..38
          0: L_CURLY@7..9 "{" [] [Whitespace(" ")]
          1: JS_NAMED_IMPORT_SPECIFIER_LIST@9..36
            0: JS_SHORTHAND_NAMED_IMPORT_SPECIFIER@9..15
              0: (empty)
              1: JS_IDENTIFIER_BINDING@9..15
                0: IDENT@9..15 "inject" [] []
            1: COMMA@15..17 "," [] [Whitespace(" ")]
            2: JS_SHORTHAND_NAMED_IMPORT_SPECIFIER@17..27
              0: (empty)
              1: JS_IDENTIFIER_BINDING@17..27
                0: IDENT@17..27 "Injectable" [] []
            3: COMMA@27..29 "," [] [Whitespace(" ")]
            4: JS_SHORTHAND_NAMED_IMPORT_SPECIFIER@29..36
              0: (empty)
              1: JS_BOGUS_BINDING@29..36
                0: IDENT@29..36 "inject" [] [Whitespace(" ")]
          2: R_CURLY@36..38 "}" [] [Whitespace(" ")]
        2: FROM_KW@38..43 "from" [] [Whitespace(" ")]
        3: JS_MODULE_SOURCE@43..58
          0: JS_STRING_LITERAL@43..58 "'@angular/core'" [] []
        4: (empty)
      2: SEMICOLON@58..59 ";" [] []
  4: EOF@59..59 "" [] []

```


* snap file from the tests

![Image](https://github.com/user-attachments/assets/6e4bb622-1e68-417d-a2b8-b81e5ff5405f)

**Comment by ematipico** [comment]
@amasotti That's because the playground is a mirror of 1.9.4. You should use `next.biomejs.dev`, or `main`,  as explained in the issue

**Comment by ematipico** [comment]
Definitely weird

**Comment by siketyan** [comment]
It seems a playground's issue. I confirmed Biome returns correct diagnostics.

<img width="1008" alt="Image" src="https://github.com/user-attachments/assets/03eff614-bef8-4d23-b796-ac77d7117c4f" />
